controllers/configurationpolicy_controller.go

```diff
@@ -1041,9 +1041,9 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc policyv1.Confi
 		nonCompliantObjects := map[string]map[string]interface{}{}
 		compliantObjects := map[string]map[string]interface{}{}
 		enforce := strings.EqualFold(string(plc.Spec.RemediationAction), string(policyv1.Enforce))
-		kind := ""
+		kind := templateObjs[indx].kind
```

Comment:
we might need to have a check here so the controller doesn't hit a panic if kind is not specified

Reply:
okie! good idea